gphoto2cffi/backend.py

```diff
@@ -131,6 +123,22 @@ def __getattr__(self, name):
         else:
             return val
 
+    def _logging_callback(self, level, domain, message, data):
```

Comment:
No it doesn't.  It's an artifact of older code.

Originally I had the `if self.logging_enabled` check in there, but then I moved it up to the wrapper in `__init__()`.  We have three options:

 1. Leave it be and merge the pull request.  No harm done
 2. Move the `if self.logging_enabled` back to the `_logging_callback()` function.  This would make it a little more clean if we wanted to add some other features to `LibraryWrapper` that effect logging
 3. Move the body of `_logging_callback()` to `_logging_callback_wrapper()` in `__init__()`.

Which one do you want?